### PR TITLE
Fixed incorrect package declarations CommonSchema and VaultSchema

### PR DESCRIPTION
**Pull request by shamsasari**

### CODE REVIEW

File: core/src/main/kotlin/net/corda/core/schemas/testing/DummyDealStateSchemaV1.kt [hunk 1]
```diff
@@ -21,5 +23,5 @@ object DummyDealStateSchemaV1 : net.corda.core.schemas.MappedSchema(schemaFamily
             @javax.persistence.Transient
             val uid: net.corda.core.contracts.UniqueIdentifier
 
-    ) : net.corda.node.services.vault.schemas.jpa.CommonSchemaV1.LinearState(uid = uid)
```
Comment (josecoll) [inline core/src/main/kotlin/net/corda/core/schemas/testing/DummyDealStateSchemaV1.kt]: Any reason we can't use the class name (and import the package)?

File: core/src/main/kotlin/net/corda/core/schemas/CommonSchema.kt [hunk 2]
```diff
@@ -36,15 +33,15 @@ object CommonSchemaV1 : MappedSchema(schemaFamily = CommonSchema.javaClass, vers
 
     @MappedSuperclass
     open class FungibleState(
-            /** [ContractState] attributes */
+            /** [net.corda.core.contracts.ContractState] attributes */
```
Comment (josecoll) [inline core/src/main/kotlin/net/corda/core/schemas/CommonSchema.kt]: import package, reference classname only?

File: core/src/main/kotlin/net/corda/core/schemas/CommonSchema.kt [hunk 3]
```diff
@@ -36,15 +33,15 @@ object CommonSchemaV1 : MappedSchema(schemaFamily = CommonSchema.javaClass, vers
...
             @OneToMany(cascade = arrayOf(CascadeType.ALL))
             var participants: Set<CommonSchemaV1.Party>,
 
-            /** [OwnableState] attributes */
+            /** [net.corda.core.contracts.OwnableState] attributes */
```
Comment (josecoll) [inline core/src/main/kotlin/net/corda/core/schemas/CommonSchema.kt]: ditto

File: core/src/main/kotlin/net/corda/core/schemas/CommonSchema.kt [hunk 4]
```diff
@@ -36,15 +33,15 @@ object CommonSchemaV1 : MappedSchema(schemaFamily = CommonSchema.javaClass, vers
...
             @OneToOne(cascade = arrayOf(CascadeType.ALL))
             var ownerKey: CommonSchemaV1.Party,
 
-            /** [FungibleAsset] attributes
+            /** [net.corda.core.contracts.FungibleAsset] attributes
```
Comment (josecoll) [inline core/src/main/kotlin/net/corda/core/schemas/CommonSchema.kt]: ditto

**Comment by shamsasari** [comment]
Yeah we can do that

File: core/src/main/kotlin/net/corda/core/schemas/testing/DummyDealStateSchemaV1.kt (hunk 1 above)
Comment (shamsasari) [inline core/src/main/kotlin/net/corda/core/schemas/testing/DummyDealStateSchemaV1.kt]: That is what's happened.

**Comment by shamsasari** [comment]
Comments addressed.

**Pull request merged**